vtysh/vtysh_config.c

```diff
@@ -254,6 +295,9 @@ void vtysh_config_parse_line(void *arg, const char *line)
 	 * generated
 	 * dynamically in vtysh_config_dump() */
 	case '!':
```

Comment:
Err.  Using the `!` comment to exit/clear keychain node status is not a sensible way to do this.  PR #9331 can probably do this correctly?

Reply:
Will take it up after that commit is merged. 
If its merged before this PR. I will incorporate those changes.

Reply:
It's wrong with the `!` so you can either copy the change to `lib/keychain.c` from #9331 into this PR or wait until #9331 is merged.  "Fixing it later" is not OK.

Reply:
I recently submitted a PR fixing this duplicated key chain config, please take a look: https://github.com/FRRouting/frr/pull/9396

Reply:
By the way, `key X` node already prints `exit` so there's no need to copy anything from #9331.

Reply:
going to revert the changes. Since #9396 is better then this fix. 
Thanks